src/registers/far_el3.rs

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Copyright (c) 2018-2023 by the author(s)
+//
+// Author(s):
+//   - Andre Richter <andre.o.richter@gmail.com>
+//   - rmsyn <rmsynchls@gmail.com>
+
+//! Fault Address Register - EL3
+//!
+//! Holds the faulting Virtual Address for all synchronous Instruction or Data Abort, PC alignment
+//! fault and Watchpoint exceptions that are taken to EL3.
```

Comment:
No need to mention Watchpoint exceptions

```suggestion
//! fault exceptions that are taken to EL3.
```

![Screenshot 2023-07-18 at 4 05 46 PM](https://github.com/rust-embedded/aarch64-cpu/assets/52972368/aff49504-cebd-4cbd-bcc7-27813b6386d6)

Reply:
Ok, that makes sense. Just added what was in the version that I had. I'll remove it.